action.yml

```diff
@@ -13,6 +13,10 @@ inputs:
     description: "The version of GFlows to install (default: latest)"
     required: false
     default: latest
+  github-api-url:
+    description: 'configure github API URL. Default `$GITHUB_API_URL`'
```

Comment:
If I'm reading your latest commit correctly, I think this description should be updated?

```suggestion
    description: 'configure github API URL. Default `https://api.github.com`'
```

Reply:
oh yea absolutely :)